src/api/public/apidocs-new/components/responses/unknown_parameter.yaml

```diff
@@ -0,0 +1,10 @@
+description: Unknown parameter
+content:
+  application/xml; charset=utf-8:
+    schema:
+      $ref: '../schemas/api_response.yaml'
+    examples:
+      Unknown parameter:
+        value:
+          code: unknown_parameter
+          summary: "unknown parameter 'limit'"
```

Comment:
I would not create this file if this component is going to be used only once, or there isn't any plan to reuse it.